src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
## Optional
- Please if you start to disable the linters by now, then it means you are going to do so for the rest of the milestone. Kindly learn how to solve the linters errors; It will make you become a great programmer. Don't deable